packages/core/src/test/shared/utilities/testUtils.test.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import assert from 'assert'
+import { createExecutableFile, createTestWorkspaceFolder, copyEnv } from '../../testUtil'
+import path from 'path'
+import { fs } from '../../../shared'
+import { isWin } from '../../../shared/vscode/env'
+import { tryRun } from '../../../shared/utilities/pathFind'
+
+describe('copyEnv', function () {
+    it('modifies the node environment variables (Non-Windows)', function () {
+        // PATH returns undefined on Windows.
+        if (isWin()) {
+            this.skip()
```

Comment:
PATH should not be undefined on Windows. Even if it did, if I'm reading the test correctly, it looks like it should still pass